src/sql/workbench/contrib/query/browser/flexibleSash.ts

```diff
@@ -95,7 +95,10 @@ export class HorizontalFlexibleSash extends Disposable implements IHorizontalSas
 	}
 
 	public getHorizontalSashWidth?(): number {
-		return this.dimension.width;
```

Comment:
maybe this? : this.dimension> this.dimension.width : 0;

Reply:
Or just `return this.dimension?.width ?? 0`

Reply:
Seems easier that way